README.md

```diff
@@ -5,29 +5,48 @@ You can get started right away with [Microk8s](https://microk8s.io/) - the might
 You can install MicroK8s on your Ubuntu laptop, workstation, or nodes in your workgroup or server cluster with just one command - ```snap install microk8s --classic```. Learn more at [microk8s.io](https://microk8s.io/).
 
 ## Spark-client Snap
-The ***spark-client*** snap includes the scripts [spark-submit](/docs/submit.md), [spark-shell](/docs/shell.md), [pyspark](/docs/pyspark.md) and other tools for managing ***Apache Spark*** jobs for ***Kubernetes***.
+The ***spark-client*** snap includes the scripts to run jobs via [spark-submit](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953), 
+or using [interactive shells](https://discourse.charmhub.io/t/spark-client-snap-tutorial-interactive-mode/8954), and other tools for 
+managing ***Apache Spark*** jobs for ***Kubernetes***.
 
 ## Setup
-The spark-client snap simplifies the setup to run Spark jobs against your Kubernetes cluster. Please follow the instructions in the [setup](/docs/setup.md) section to get started.
+The spark-client snap simplifies the setup to run Spark jobs against your Kubernetes cluster. Please 
+follow the instructions in the [setup](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8952) section to get started.
 
-Do check out the section on [config resolution](/docs/config.md) to understand how [spark-submit](/docs/submit.md) actually resolves the configuration properties coming from a diverse set of available sources.
+Do check out the section on [config resolution](https://discourse.charmhub.io/t/spark-client-snap-explanation-hierarchical-configuration-handling/8956) to understand 
+how [spark-submit](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953) actually resolves the configuration properties coming 
+from a diverse set of available sources.
 
 ## Play!
-Once the [setup](/docs/setup.md) is complete, please follow the Spark job [submission guide](/docs/submit.md) to validate and start utilizing your 
+Once the [setup](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8951) is complete, 
+please create a Spark service account using the [CLI](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8952) and follow the Spark 
```

Comment:
I would remove "please" here and just be imperative

Reply:
I like to be nice to people :joy: 

I have removed "please" throughout :D